presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/PermissionMetadata.java

```diff
@@ -54,6 +56,6 @@ public boolean isGrantOption()
 
     public HivePrivilegeInfo toHivePrivilegeInfo()
     {
-        return new HivePrivilegeInfo(permission, grantOption);
+        return new HivePrivilegeInfo(permission, grantOption, new PrestoPrincipal(USER, "admin"));
```

Comment:
This isn't necessarily accurate though, right? And it wouldn't even work in one given session, you wouldn't see the actual grantor